install/installer/third_party/charts/rabbitmq/values.yaml

```diff
@@ -31,7 +31,6 @@ rabbitmq:
     enabled: true
     existingSecret: load-definition
   extraConfiguration: |
-    load_definitions = /app/load_definition.json
```

Comment:
Shouldn't this be `/gitpod-config/load_definition.json` then? as it should follow the same in  `helm.go` render so that we have testing parity? 🤔

Reply:
Not that I can see. I put on this explaining my findings:

> Not needed as far as I can see - this is set in the Helm chart. This one takes precedence, which breaks our config. See [docs](https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq#load-custom-definitions) and "NOTE: Loading a definition will take precedence over any configuration done through [Helm values](https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq#parameters)."

Reply:
Got it